TPP.Core/Commands/Definitions/BettingCommands.cs

```diff
@@ -0,0 +1,74 @@
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using TPP.ArgsParsing.Types;
+using TPP.Match;
+using TPP.Persistence.Models;
+
+namespace TPP.Core.Commands.Definitions
+{
+    public class BettingCommands : ICommandCollection
+    {
+        private readonly Func<IBettingPeriod<User>?> _bettingPeriodProvider;
+
+        /// <summary>
+        /// Command collection for all betting related commands.
+        /// </summary>
+        /// <param name="bettingPeriodProvider">Provides the current betting period,
+        /// which may be null if there currently is no betting period running.</param>
+        public BettingCommands(Func<IBettingPeriod<User>?> bettingPeriodProvider)
+        {
+            _bettingPeriodProvider = bettingPeriodProvider;
+        }
+
+        public IEnumerable<Command> Commands => new[]
+        {
+            new Command("bet", Bet)
+            {
+                Description = "Chat only: bet on PBR using Pokéyen. Arguments: <team> <amount or percent>"
+            }
+        };
+
+        public async Task<CommandResult> Bet(CommandContext context)
+        {
+            if (context.Message.MessageSource != MessageSource.Chat)
```

Comment:
again see above, though I realise this is heavily dependant on the modes implementation